database/stats.go

```diff
@@ -8,17 +8,41 @@ import (
 )
 
 const statsQuery = `
-SELECT COALESCE(SUM(serviceFee + onchainFee), 0), COALESCE(SUM(expectedAmount), 0), COUNT(*)
-FROM (SELECT serviceFee, onchainFee, expectedAmount, isAuto, createdAt, 'submarine' type, tenantId
+SELECT COALESCE(SUM(COALESCE(serviceFee, 0) + COALESCE(onchainFee, 0)), 0),
+       COALESCE(SUM(CASE WHEN state == 1 THEN amount END), 0),
+       COUNT(*),
+       COUNT(CASE WHEN state == 1 THEN 1 END)
+FROM (SELECT state,
+             serviceFee,
+             onchainFee,
+             expectedAmount amount,
+             isAuto,
+             createdAt,
+             'submarine'    type,
```

Comment:
The indentation of this statement is weird

Reply:
its goland formatting, not sure

Reply:
Still, look at it in github. Super weird